model/job/trigger_cron.go

```diff
@@ -29,10 +32,31 @@ func NewCronTrigger(infos *TriggerInfos) (*CronTrigger, error) {
 	}, nil
 }
 
-// NewEveryTrigger returns an new instance of CronTrigger given the specified
+// NewEveryTrigger returns a new instance of CronTrigger given the specified
 // options as @every.
 func NewEveryTrigger(infos *TriggerInfos) (*CronTrigger, error) {
-	schedule, err := parser.Parse("@every " + infos.Arguments)
+	schedule, err := cronParser.Parse("@every " + infos.Arguments)
+	if err != nil {
+		return nil, ErrMalformedTrigger
+	}
+	return &CronTrigger{
+		TriggerInfos: infos,
+		sched:        schedule,
+		done:         make(chan struct{}),
+	}, nil
+}
+
+// NewWeeklyTrigger returns a new instance of CronTrigger given the specified
+// options as @weekly. It will take a random day/hour in the possible range to
+// spread the triggers from the same app manifest.
+func NewWeeklyTrigger(infos *TriggerInfos) (*CronTrigger, error) {
+	spec, err := periodicParser.Parse(infos.Arguments)
+	if err != nil {
+		return nil, ErrMalformedTrigger
+	}
+	seed := infos.Domain + "/" + infos.WorkerType
```

Comment:
Yes

Reply:
It is probably better to avoid that, I have changed it in https://github.com/cozy/cozy-stack/pull/3460/commits/732eb6ef1a7baa1098d9b41013cb92579eec49b6